src/e2e/tests/1-create-birth.ts

```diff
@@ -0,0 +1,48 @@
+import path from 'node:path'
+
+import execa from 'execa'
+
+import { Leon } from '../../services/Leon.js'
+import { isExistingFile } from '../../utils/isExistingFile.js'
+
+interface Options {
+  useDocker?: boolean
+}
+
+export const test1 = (options: Options = {}): void => {
+  const { useDocker = false } = options
+
+  it('leon create birth', async () => {
+    const commandOptions = useDocker ? ['--docker'] : []
+    expect(await isExistingFile(Leon.DEFAULT_BIRTH_PATH)).toBeFalsy()
```

Comment:
Oh ok, I didn't know that. 
Indeed according to jest documentation about [`.toBeFalsy`](https://jestjs.io/docs/expect#tobefalsy) and [`.toBeTruthy`](https://jestjs.io/docs/expect#tobetruthy), it is checking `false`, `0`, `''`, `null`, `undefined`, and `NaN`, but in our case, we just need to check `false`.
Anyway, I guess TypeScript would catch it anyway if, for example, we would return `0` instead of `false`.

But I agree strict check is always better.
I search in the whole codebase, the occurrences of `toBeTruthy` and replaced it with `toBe(true)` and `toBeFalsy`  with `toBe(true)`. :+1: